File: src/main/frontend/frontend/src/base/Message.js
export default function Message(props)
{

    var date = props.date;
    var sender = props.sender;
    var content = props.content;

    return(
        <tr>
            <td>{date}</td>
            <td>{sender}</td>
            <td>{content}</td>
        </tr>
    )
}

File: src/main/java/com/StreletsA/chat/model/Message.java
package com.StreletsA.chat.model;


public class Message {

    private String date;
    private String sender;
    private String content;

    public Message(String date, String sender, String content){
        this.date = date;
        this.sender = sender;
        this.content = content;
    }

    public String getSender() {
        return sender;
    }

    public String getContent() {
        return content;
    }

    public String getDate() {
        return date;
    }
}
